drake/doc/developers.rst

```diff
@@ -43,6 +43,47 @@ available in the superbuild.
 class method runs as a unit test, and fails if it times out or calls
 ``error``. For more information, see ``drake/util/unitTest.m``.
 
+Supported Configurations
+========================
+
+The following table shows the configurations and platforms that Drake
+officially supports. Supported configurations are tested in continuous
+integration. All other configurations are provided on a best-effort basis.
+
++------------------------+-----------------+-------------------+---------+
+| Operating System       | Compiler        | Superbuild Deps   | Build   |
++========================+=================+===================+=========+
+| Ubuntu 14.04 LTS       | GCC 4.9         | Default           | Debug   |
```

Comment:
Is it worth saying that we only support makefile builds, rather than the full range of CMake-supported generators? Same for OS X.